src/lib/PostgrestFilterBuilder.ts

```diff
@@ -4,7 +4,15 @@ import PostgrestTransformBuilder from './PostgrestTransformBuilder'
  * Filters
  */
 
-const cleanFilterArray = <T>(filter: T[keyof T][]) => filter.map((s) => `${s}`).join(',')
+const cleanFilterArray = <T>(filter: T[keyof T][]) =>
+  filter
+    .map((s) => {
+      // handle postgrest reserved characters
+      // https://postgrest.org/en/v7.0.0/api.html#reserved-characters
+      if (typeof s === 'string' && new RegExp('[,.():]').test(s)) return `"${s}"`
```

Comment:
Just `()` and `,` need quoting inside `in`. Quoting `:` and `.` is only needed when quoting column names.

```suggestion
      if (typeof s === 'string' && new RegExp('[,()]').test(s)) return `"${s}"`
```